imap/imapd.c

```diff
@@ -11198,7 +11198,13 @@ struct xfer_header {
     struct xfer_item *items;
 };
 
-static int xfer_mupdate(int isactivate,
+enum {
+    MUPDATE_ACTIVATE,
+    MUPDATE_DEACTIVATE,
+    MUPDATE_DELETE
+};
+
+static int xfer_mupdate(int op,
```

Comment:
I think I would name this enum, and then use it as the op type instead of `int`.  It'll get us a few extra compiler protections if the function knows it's an enumerated type and not just some number.

Something like:

```
enum mupdate_op {
   ...
};

static int xfer_mupdate(enum mupdate_op op,
```

Reply:
Done